database/factories/ModelFactory.php

```diff
@@ -205,7 +205,8 @@
 $factory->define(GroupType::class, function (Generator $faker) {
     return [
         'name' => 'National ' . Str::title($faker->unique()->jobTitle) . ' Society',
-        'filter_by_state' => false,
+        'filter_by_location' => true,
+        'filter_by_state' => true,
```

Comment:
why is the `filter_by_state` needed here?

Reply:
This is the change that corresponds to the 3rd misc item in the PR description:
> Modifies the Group factory to populate with a location, so that running the VoterRegistrationSeeder (#1086) will generate action stats with valid location values and allow filtering by location.